README.md

```diff
@@ -1,58 +1,73 @@
-# helium-miner-software
-Software for Nebra Helium Miners
+# Helium Miner Software
 
-This dockerfile creates 6 containers.
+This repository includes the main [docker-compose.yml](https://github.com/NebraLtd/helium-miner-software/blob/master/docker-compose.yml) file that powers the Nebra miners.
 
-## diagnostics
+The `docker-compose.yml` file is pushed to [Balena](https://www.balena.io/) (using GitHub Actions), which in turn pulls down the various Docker images outlined below.
 
-Basic container that runs a webserver to provide diagnostics information & manufacutring tools
+There are currently six different services running within this device, which are all outlined below.
 
-https://github.com/NebraLtd/hm-diag
+## Diagnostics
 
-## packet-forwarder
+Repo: [github.com/NebraLtd/hm-diag](https://github.com/NebraLtd/hm-diag)
 
-The container that has the code that configures the packet forwarder's region and starts the radio module.
+The diagnostics container is designed for local troubleshooting. It runs a local web server that displays various diagnostics data.
 
-https://github.com/NebraLtd/hm-pktfwd
+## Packet Forwarder
 
-## gateway-config
+Repo: [github.com/NebraLtd/hm-pktfwd](https://github.com/NebraLtd/hm-pktfwd)
 
-The container that has the code to provide the Bluetooth LE to allow the hotspot to be configured via the Helium App.
+This container is responsible for configuring packet forwarder's region and starts the radio module.
 
-https://github.com/NebraLtd/hm-config
+## Gateway Config
 
-## helium-miner
+Repo: [github.com/NebraLtd/hm-config](https://github.com/NebraLtd/hm-config)
 
-The Helium Miner with the required configuration files added.
+This container is responsible for the device onboarding and provides the Bluetooth LE to allow the hotspot to be configured via the Helium App.
```

Comment:
Well it's partially responsible, as much of the onboarding side is done on Helium side. This is as much for wi-fi configuration and following through the steps.